cnudie/retrieve.py

```diff
@@ -163,17 +163,12 @@ def writeback(
 
     def lookup(
         component_id: ocm.ComponentIdentity,
-        ctx_repo: ocm.OcmRepository=None,
```

Comment:
Please also adjust type definition of `ComponentDescriptorLookupById` (ditto for `cnudie.retrieve_async.ComponentDescriptorLookupById`).

Reply:
again: good catch (however, the signature was not accurate before this change :-/)